OpenUpSA/wazimap-ng-ui
# handling missing chart data

## Description
The assumption of every `children` key of subindicators would have `count` property caused some charts to not work. Added some controls to fix that

## Related Issue
https://trello.com/c/QqmElVl5/495-ensure-that-the-fe-assumes-a-0-for-count-where-data-is-missing-in-order-to-not-break-other-aspects-of-the-site


## How to test it locally
* Navigate to a geography that has missing count property, for example `/#geo:EC` on `sifar-wazi.openup.org.za` (this could be done by hardcoding `hostname` parameter in `index.js`).
* Check if the charts like `Marital Status` are working.
* Note that some charts like `Older Adults Population` are still not working because _ALL_ the count properties are missing so they are all assigned 0.

## Screenshots
![image](https://user-images.githubusercontent.com/53019884/94813458-b057fe80-0400-11eb-9e9d-09a3231c8285.png)
![image](https://user-images.githubusercontent.com/53019884/94813577-db425280-0400-11eb-83b5-97eb7842da65.png)


## Changelog

### Added

### Updated
* Updated `chart.js` to add necessary checks

### Removed


## Checklist


### Pull Request


### Commits


### Code Quality


### Testing
